### ISSUE TITLE
Use human dataset， but got bad performance

### ISSUE THREAD

**Issue by zoe218** [CoinCheung/BiSeNet#114]
Hello, thanks for your  great reproduce! 
I try to use my own dataset to train this model, but the performance is really bad, like:

![image](https://user-images.githubusercontent.com/24896757/103186171-6fd22900-48fa-11eb-9c69-4de7efce84a7.png)

I change the nn.pixelshuffle to interpolate; the loss is celoss(this dataset only contains 2 classes: human, background); batch size is 16, 160 epoch.
Do you have some advice to improve the performance?

**Comment by zoe218** [comment]
The image input size is [224, 288]

**Comment by CoinCheung** [comment]
Hi,

Please patch the details of how you modified the original code. Also please note that there is label map in the dataset class. Just looking at the image, I feel it is more likely that your code is not correct. Besides, if you change the dataset, you might need to re-adjust the training hyper-parameters.

**Comment by novioleo** [comment]
the right side segmentation is better than the left side

**Comment by CoinCheung** [comment]
I doubt the problem is about how you prepare your training set rather than pixeshuffle. Did you observe same problem when you resume  pixelshuffle ?

**Comment by zoe218** [comment]
@CoinCheung Thanks so much! I'll check the training set. But the V2 is not faster the V1, right?

**Comment by CoinCheung** [comment]
Yes, since v2 uses a lot of separate conv which does not access memory as efficiently as plain conv. The memory latency is thus larger. That is why v2 has less FLOPs than v1, but is still slower.
